src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,19 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private boolean frontBlade;
```

Comment:
Consider adding access modifiers for the 'frontBlade' field to encapsulate the class field properly as per the checklist requirement: 'Don't forget about access modifiers for your custom fields in `Excavator`, `Truck` and `Bulldozer` classes.'